native/src/components/FilterToggle.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components/native'
+
+import { ExpandIcon, ShrinkIcon } from '../assets'
+import Icon from './base/Icon'
+import Text from './base/Text'
+
+const StyledText = styled(Text)`
+  font-weight: bold;
+  padding: 5px;
+`
+const StyledButton = styled.TouchableOpacity`
+  display: flex;
+  flex-direction: ${props => (props.theme.contentDirection === 'rtl' ? 'row-reverse' : 'row')};
+  align-items: center;
+  gap: 5px;
+  justify-content: center;
+  align-self: ${props => (props.theme.contentDirection === 'rtl' ? 'flex-end' : 'flex-start')};
```

Comment:
which two lines? flex-direction & align-self ?
both of these have a purpose I'm not getting what you are trying to say here...

Reply:
I kept `align-self` just because DateSection (parent) has `align-items: center;` & ` flex-direction: column;` to keep them in the center so StyledButton I need it to the left or the right depending on the language.